ts/annotations/controllable/ControllablePath.ts

```diff
@@ -148,32 +148,31 @@ merge<Highcharts.AnnotationControllablePath, Partial<Highcharts.AnnotationContro
                 point = points[0],
                 position = showPath && this.anchor(point).absolutePosition,
                 pointIndex = 0,
-                dIndex = 2,
-                command;
-
-            d = (position && ['M', position.x, position.y]) as any;
-
-            while (++pointIndex < len && showPath) {
-                point = points[pointIndex];
-                command = point.command || 'L';
-                position = this.anchor(point).absolutePosition;
-
-                if (command === 'Z') {
-                    (d as any)[++dIndex] = command;
-                } else {
-                    if (command !== points[pointIndex - 1].command) {
-                        (d as any)[++dIndex] = command;
+                command,
+                d: Highcharts.SVGPathArray = [];
+
+            if (position) {
+                d.push(['M', position.x, position.y]);
+
+                while (++pointIndex < len && showPath) {
+                    point = points[pointIndex];
+                    command = point.command || 'L';
+                    position = this.anchor(point).absolutePosition;
+
+                    if (command === 'M') {
+                        d.push([command, position.x, position.y]);
+                    } else if (command === 'L') {
+                        d.push([command, position.x, position.y]);
```

Comment:
One should think so, but apparently TypeScript gets confused as to whether this is a `SVGPathLineTo` or `SVGPathMoveTo`. If you have a proposal to fix that, it would be great.
`Argument of type '["L" | "M", number, number]' is not assignable to parameter of type 'SVGPathSegment'`

Reply:
I see. I did just noticed, that it was (kind of) working here: https://github.com/highcharts/highcharts/blob/022878f65ff240f81a5e107818dfcb7b45ac9264/ts/parts-more/RadialAxis.ts#L766, but that is probably because of `path` beeing of type `Array<any>`.

Reply:
It was `xy` that was `any`. Fixed that now.